sprint2/style.css

```
@@ -0,0 +1,210 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&dislay=swap");
+*
+{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family:'Poppins',sans-serif;
+}
+body
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 1300px;
+    background: #f5f5f5;
+}
+.container
+{
+    position: relative; 
+    width: 100%;
+    max-width: 1200px;
+    min-height: 1650px ;
+    background: #fff;
+    margin: 50px;
+    box-shadow: 0 35px 55px rgba(0,0,0,0.1);
+}
+.top
+{
+    position: relative;
+    width: 100%;
+    height: 400px;
+    background: linear-gradient(45deg, #ceb7a1, #7a2f70);
+    padding: 50px;
+}
+h2
+{
+    font-size: 55px ;
+    color: #ffffff;
+    text-transform: uppercase;
+    line-height: 35px;
+    letter-spacing: 5px;
+}
+h2 span
+{
+    font-size: 0.4em;
+    font-weight: 300;
+}
+.group 
+{
+    position: relative;
+    display: grid;
+    gap: 90px;
+    width: 100%;
+    grid-template-columns: 2fr 3fr;
+    margin-top: 22px;
+}
+.grup .imgBx
+{
+    position: relative;
+    width: 100%;
+}
+.grup .imgBx img
+{
+    width: 100%;
+} 
+.group p
+{
+    color: #fff;
+    font-size: 1.05em;
+}
+.bottom
+{
+    position: relative;
+    display: grid;
+    margin: 50px 50px 0px;
+    gap: 40px;
+    grid-template-columns: 2fr 3fr;
+}
+.bottom .left_side
+{
+    margin-top: 50px;
+}
+h4
+{
+    font-size: 20px;
+    text-transform: uppercase;
+    letter-spacing: 0.25em;
+    color: #fff;
+    background: #968371;
+    font-weight: 600;
+    padding: 3px 15px;
+    margin-top: 40px;
+    display: inline-flex;
+}
+.contactInfo
+{
+    position: relative;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    margin: 20px 0;
+}
+.contactInfo li
+{
+    list-style: none;
+    display: flex;
+    gap: 10px;
+}
+.contactInfo li .icon
+{
+    color: #7c4274;    
+}
+.contactInfo li .text
+{
+    color: #52264c;
+}
+.skills
+{
+    position: relative;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+    gap: 18px;
+    margin: 20px 0;
+}
+.skillsInfo li
```

```suggestion
.skills-info li
```
(optional) Recomand toate numele de clasa sa fie in stilul kebab-case, adica cu liniute in loc de underscore si fara litere mari.